Mathlib/Algebra/Field/Defs.lean

```diff
@@ -113,7 +113,7 @@ class DivisionRing (α : Type*)
   protected ratCast_def (q : ℚ) : (Rat.cast q : α) = q.num / q.den := by intros; rfl
   /-- Scalar multiplication by a rational number.
 
-  Set this to `qsmulRec _` unless there is a risk of a `Module ℚ _` instance diamond.
+  Write `qsmul := _` unless there is a risk of a `Module ℚ _` instance diamond.
```

Comment:
Let's add "(which is unified to `(Rat.cast \. * \.)` automatically thanks to the default proof of `qsmul_def`)"

Reply:
Fine to do in a follow-up since this is already in the queue, and your change is still better than no change.

It looks like we should just delete `qsmulRec` altogether?

Reply:
I've deleted `qsmulRec` and updated the docstring